lib/rory/request_parameter_logger.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'parameter_filter'
+
+module Rory
+  class RequestParameterLogger
+
+    def initialize(app, logger=nil, filters=[:password, :tin, :ssn, :social_security_number, :file_attachment])
```

Comment:
This seems like a strangely specific list to put in here.  I like `:password`, but the rest feel like random guesses about how a client would name these things, or even if they'd want to filter them, just based on our own names.